apiv6/generate_server.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Generate golang server stub from OpenAPI 3.0.0 spec
```

Comment:
I see following files on the branch:
```bash
apiv6/localhost_apiserver
├── api
│   └── openapi.yaml
├── CloudExportStorage.json
├── go
│   ├── api_cloud_export_admin_service.go
│   ├── api_cloud_export_admin_service_service.go
│   ├── api.go
│   ├── helpers.go
│   ├── impl.go
│   ├── logger.go
│   ├── model_cloud_exportv202101beta1_status.go
│   ├── model_googlerpc_status.go
│   ├── model_protobuf_any.go
│   ├── model_v202101beta1_aws_properties.go
│   ├── model_v202101beta1_azure_properties.go
│   ├── model_v202101beta1_bgp_properties.go
│   ├── model_v202101beta1_cloud_export.go
│   ├── model_v202101beta1_cloud_export_type.go
│   ├── model_v202101beta1_create_cloud_export_request.go
│   ├── model_v202101beta1_create_cloud_export_response.go
│   ├── model_v202101beta1_gce_properties.go
│   ├── model_v202101beta1_get_cloud_export_response.go
│   ├── model_v202101beta1_ibm_properties.go
│   ├── model_v202101beta1_list_cloud_export_response.go
│   ├── model_v202101beta1_patch_cloud_export_request.go
│   ├── model_v202101beta1_patch_cloud_export_response.go
│   ├── model_v202101beta1_update_cloud_export_request.go
│   ├── model_v202101beta1_update_cloud_export_response.go
│   ├── repo.go
│   └── routers.go
├── main.go
├── README.md
├── run_server.sh
└── run_swagger_ui.sh
```

So we have generated files for cloudexport, but no for synthetics. As we are trying to minimize checked in generated files, could you delete those for cloudexport that are already checked in? Or are you planning to do that in separate PR?

Reply:
Ok, I see that this code is only partially generated so we need to check that in. Resolved.

Reply:
localhost_apiserver is required by cloudexport provider.
Support for synthetics will be added soon.